pennylane/templates/embeddings/displacement.py

```diff
@@ -41,6 +41,58 @@ class DisplacementEmbedding(Operation):
         c (float): value of the phase of all displacement gates if ``execution='amplitude'``, or
             the amplitude of all displacement gates if ``execution='phase'``
 
+    Example:
```

Comment:
One detail: we'd like the `Example` section to come after `Raises`.
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:
```